### DISCUSSION TITLE
Do you have a plan to drop python < 3.7

### DISCUSSION THREAD

**Discussion by mangin** [pypa/virtualenv#2526] [Q&A] [answered]
Hey there,
I tried to find any discussion about it in your issues, but didn't find.

The reason why maybe we should do it:
1. python < 3.7 are not maintained anymore.
2. and some of libraries like `wheel` stops to support deprecated pythons.

https://github.com/pypa/virtualenv/blob/ebbccecdb65725025dfc8f4af504d7de3a770e95/src/virtualenv/seed/wheels/embed/__init__.py#L40

**Comment by gaborbernat** [comment, marked as answer]
3.7 EOL is 2023-06-27, which is not yet here. But virtualenv supports EOL pythons for another 1.5 years. So no plans to drop it anytime soon.

**Reply by mangin** [reply]
>virtualenv supports EOL pythons for another 1.5 years.
Good to know, thx

What's about python 3.6, 3.5 and 2.7?
https://endoflife.date/python

if i understand your life cycle correctly you can drop 3.5 (Support ended 2 years and 6 months ago ) and 2.7 (Support ended 3 years ago ). But you still should support 3.6 for 3 months (Ended 1 year and 2 months ago )

**Reply by gaborbernat** [reply]
Yes. We haven't dropped support for those purely because I didn't get to it. But will do at some point soon-ish.

**Comment by esafak** [comment]
This is done.
